x-pack/plugins/enterprise_search/public/applications/workplace_search/components/layout/personal_dashboard_layout/personal_dashboard_layout.tsx

```diff
@@ -6,44 +6,67 @@
  */
 
 import React from 'react';
+import { useRouteMatch } from 'react-router-dom';
 
-import { EuiPage, EuiPageSideBar, EuiPageBody, EuiCallOut } from '@elastic/eui';
+import { useValues } from 'kea';
 
-import { AccountHeader } from '..';
+import {
+  EuiPage,
+  EuiPageSideBar,
+  EuiPageBody,
+  EuiPageContentBody,
+  EuiCallOut,
+  EuiSpacer,
+} from '@elastic/eui';
 
+import { FlashMessages } from '../../../../shared/flash_messages';
+import { HttpLogic } from '../../../../shared/http';
+import { SetWorkplaceSearchChrome } from '../../../../shared/kibana_chrome';
+import { BreadcrumbTrail } from '../../../../shared/kibana_chrome/generate_breadcrumbs';
+import { Loading } from '../../../../shared/loading';
+
+import { PERSONAL_SOURCES_PATH, PERSONAL_SETTINGS_PATH } from '../../../routes';
 import { PRIVATE_DASHBOARD_READ_ONLY_MODE_WARNING } from '../../../views/content_sources/constants';
+import { AccountHeader, AccountSettingsSidebar, PrivateSourcesSidebar } from '../index';
 
 import './personal_dashboard_layout.scss';
 
 interface LayoutProps {
-  restrictWidth?: boolean;
-  readOnlyMode?: boolean;
-  sidebar: React.ReactNode;
+  isLoading?: boolean;
+  pageChrome?: BreadcrumbTrail;
 }
 
 export const PersonalDashboardLayout: React.FC<LayoutProps> = ({
```

Comment:
One more thought - I wonder if we should rename this `PersonalDashboardPageTemplate` to match, or nah 🤔

Reply:
Yeah, I was thinking the same. But seriously feel free to nah. 😄 
We need to do a bunch of renaming in our codebase anyway.

Reply:
Haha I'm happy punting it for now. Someday! 🌱